tests/functional/formatter.sh

```diff
@@ -34,13 +34,38 @@ cat << EOF > flake.nix
 }
 EOF
 
+mkdir subflake
+cp ./simple.nix ./simple.builder.sh ./formatter.simple.sh "${config_nix}" "$TEST_HOME/subflake"
+
+cat << EOF > subflake/flake.nix
```

Comment:
I added some tests for subflakes just in case.

Reply:
Thanks, @Mic92!